docker-compose.yml

```
@@ -0,0 +1,83 @@
+version: '2'
+services:
+  zookeeper-1:
+    image: confluentinc/cp-zookeeper
+    container_name: zookeeper-1
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 3181
+      ZOOKEEPER_TICK_TIME: 2000
+      ZOOKEEPER_INIT_LIMIT: 5
+      ZOOKEEPER_SYNC_LIMIT: 2
+    ports:
+      - "23181:3181"
+  kafka-workshop-1:
+    image: confluentinc/cp-kafka
```

```suggestion
    image: confluentinc/cp-kafka:latest
```

shall we add latest tag to this ?